src/style.css

```diff
@@ -2,2 +1,83 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.container-logo {
+  display: flex;
+}
+
+.container-logo img {
+  display: block;
+}
+
+.nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  padding: 0;
+  text-align: center;
+  height: 60px;
+  line-height: 60px;
+  margin-left: 20px;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+.nav__link[data-qa='hover']:hover {
+  color: #00acdc;
+}
+
+.nav__link.nav__item {
+  display: flex;
+  align-items: center;
+  height: 100%;
 }
```

Comment:
You are using both the class `.nav__link` and the class `.nav__item` to style the navigation links. This is a good use of multiple classes to style an element.